extensions/wikia/ContentReview/ContentReview.hooks.php

```diff
@@ -112,4 +113,33 @@ public static function onArticleContentOnDiff( $diffEngine, \OutputPage $output
 
 		return true;
 	}
+
+	public static function onRawPageViewBeforeOutput( \RawAction &$rawAction, &$text ) {
```

Comment:
Objects are effectively passed by reference in PHP anyway, so no need for the `&`.